docker/singularity

```diff
@@ -33,41 +33,42 @@ From: nvidia/cuda:10.1-cudnn7-devel-ubuntu18.04
     libopenblas-dev \
     libhdf5-dev
 
-  export PYENV_ROOT=/opt/pyenv
-  export PATH="/opt/pyenv/bin:$PATH"
-  curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash
-  pyenv install 3.7.2
-  echo 'export PATH=/opt/pyenv/versions/3.7.2/bin/:$PATH' >> $SINGULARITY_ENVIRONMENT
-  export PATH=/opt/pyenv/versions/3.7.2/bin/:$PATH
+  export PYENV_ROOT=/opt/pyenv && \
+    export PATH="/opt/pyenv/bin:$PATH" && \
+    curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash && \
+    pyenv install 3.10.3 && \
+    echo 'export PATH=/opt/pyenv/versions/3.10.3/bin:$PATH' >> $SINGULARITY_ENVIRONMENT && \
+    export PATH=/opt/pyenv/versions/3.10.3/bin:$PATH
 
-  pip install torch==1.3.0
+  pip install torch==1.11.0
 
   mkdir -p $SINGULARITY_ROOTFS/tmp/sing_build_cuda
   cd $SINGULARITY_ROOTFS/tmp/sing_build_cuda
 
-  export TORCH_CUDA_ARCH_LIST="5.0 6.1"
+  echo 'export TORCH_CUDA_ARCH_LIST="5.0 6.1"' >> $SINGULARITY_ENVIRONMENT && \
+    export TORCH_CUDA_ARCH_LIST="5.0 6.1"
 
   git clone https://github.com/rusty1s/pytorch_scatter.git && \
```

Comment:
I think we should pip install from wheels here via `-f` option. WDYT?

Reply:
Installing from wheels is a good idea. Reduce any issues that might be introduced building from source/with `setup.py`. Let me play around with it a bit to make sure it works as expected.

Reply:
Super. Let me know how it goes!

Reply:
Friendly ping :)

Reply:
Hey there! I should hopefully have something by tomorrow. Sorry about the delay. Had finals and graduation which ended up sucking up all my time :( but I am free now.